scenes/props/lever/test_lever/test-lever.gd

```diff
@@ -0,0 +1,22 @@
+extends Node2D
+
+var state = "0"
+@onready var player = $"../../../Player"
+
+func ready():
+	flip_sprite()
+
+func _process(delta):
```

Comment:
this will run every frame, while we dont press "interaction" action every frame

add "Interactable" scene (`scenes/tools/Interactable.tscn`), configure its shape (make shape resource unique -> configure shape) and subscribe to event Interactable.interacted

Interactable runs only on "interaction" key press, not every frame